### PR TITLE
Add semester controller

### PR DESCRIPTION
**Pull request by BrainMaestro**
Part of #62

### CODE REVIEW

**Comment by BrainMaestro** [comment]
Approved

File: app/Http/Controllers/SemesterController.php [hunk 1]
```diff
@@ -2,59 +2,158 @@
...
     }
+
+    /**
+     * Changes the current semester to the specified one.
+     *
+     * @param $id
+     * @param bool $status
+     * @return bool
+     */
+    private function changeCurrentSemester($id, $status = true)
```
Comment (tenorz007) [inline app/Http/Controllers/SemesterController.php]: This function should probably be in a transaction.

Comment (BrainMaestro) [inline app/Http/Controllers/SemesterController.php]: Probably. But how would that work with the `return` statements.

Comment (tenorz007) [inline app/Http/Controllers/SemesterController.php]: How does the return statement affect it

Comment (BrainMaestro) [inline app/Http/Controllers/SemesterController.php]: It will abort the function before the transaction is committed

**Comment by tenorz007** [comment]
Approved!

**Comment by tenorz007** [comment]
Approved

**Pull request merged**